lib/model/database_manager.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/foundation.dart';
+import 'package:isar/isar.dart';
+import 'package:lists/model/list_model.dart';
+import 'package:lists/model/item.dart';
+
+/// DatabaseManager:
+///   - a class that creates, reads, updates, and
+///     destroys lists (see ListModel).
+class DatabaseManager {
+  static late final Isar isar;
+
+  static Future<void> init() async {
+    if (Isar.instanceNames.isEmpty) {
+      isar =
+          await Isar.open([ListModelSchema, ItemSchema], inspector: kDebugMode);
+    }
+  }
+
+  static Future<ListModel> putListModel(ListModel listModel) async {
+    // final newListModel = ListModel.fromTitle(name);
```

Comment:
What is this line for?

Reply:
Oh, that comment is residue from old code. Will clean.